.github/workflows/ca-ssnv2-test.yml

```diff
@@ -1324,3 +1324,36 @@ jobs:
         if: always()
         run: |
           docker exec pki find /var/lib/pki/pki-tomcat/logs/ca -name "debug.*" -exec cat {} \;
+
+
+      ####################################################################################################
+      # Try to install with wrong parameter foramt
+      - name: Create CA with unsupported range format
```

Comment:
Typo: foramt

We probably should run this step before the correct `pkispawn` invocation. That will simulate the actual scenario that people might encounter, i.e. run `pkispawn` with the wrong params first, then rerun it with the correct params.